packages/rest/docs.json

```diff
@@ -0,0 +1,24 @@
+{
+    "content": [
+        "index.ts",
+        "src/application.ts",
+        "src/component.ts",
+        "src/http-handler.ts",
+        "src/index.ts",
+        "src/internal-types.ts",
+        "src/parser.ts",
+        "src/promisify.ts",
+        "src/router/metadata.ts",
+        "src/router/routing-table.ts",
+        "src/sequence.ts",
+        "src/server.ts",
```

Comment:
I believe that `server.ts` is in `core`, right?

Reply:
Please also fix https://github.com/strongloop/loopback-next/blob/feature/rest/packages/core/docs.json